src/useFetchArtworks.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from "react";
+import { fetchArtworks } from "./fetchApi";
+import { GalleryCardType } from "./components/GalleryCard";
+
+const useFetchArtworks = () => {
+  const [artworks, setArtworks] = useState<GalleryCardType[]>([]);
+
+  useEffect(() => {
+    const getArtworks = async () => {
+      const data = await fetchArtworks();
+      setArtworks(data.data);
+    };
+
+    getArtworks();
+  }, []);
+
+  return artworks;
+};
```

Comment:
Your first custom hook! 🙌 

However, I suggest to put it in a sub-directory like `hooks`, to have proper order in the `src` directory.

It could also be grouped with `fetchApi.ts` in a `api` directory.